src/encoded/schemas/long_read_rna_mapping_quality_metric.json

```diff
@@ -0,0 +1,48 @@
+
+{
+    "title": "long-read RNA-seq mapping quality metric",
+    "description": "Schema for reporting long-read RNA-seq pipeline mapping quality metric",
+    "id": "/profiles/long_read_rna_mapping_quality_metric.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "required": ["step_run", "award", "lab", "full_length_non_chimeric_read_count", "mapping_rate"],
+    "additionalProperties": false,
+    "identifyingProperties": ["uuid"],
+    "mixinProperties": [
+        { "$ref": "mixins.json#/schema_version" },
+        { "$ref": "quality_metric.json#/properties" },
+        { "$ref": "mixins.json#/uuid" },
+        { "$ref": "mixins.json#/submitted"},
+        { "$ref": "mixins.json#/aliases"},
+        { "$ref": "mixins.json#/attribution" },
+        { "$ref": "mixins.json#/standard_status"},
+        { "$ref": "mixins.json#/assay" }
+   ],
+    "properties": {
+        "schema_version": {
+            "default": "1"
+        },
+        "full_length_non_chimeric_read_count": {
+            "title": "Full length non-chimeric read count",
+            "description": "Quantity of full-length reads not mapping chimerically of the genome",
+            "type": "integer",
+            "minimum": 0
+        },
+        "mapping_rate": {
+            "title": "Mapping rate",
+            "description": "Percentage of reads mapping to the genome",
```

Comment:
This description is inaccurate and leads one to think that this would be a number between 0 and 100.

Reply:
Changed to "Proportion of reads..."